gradle.properties

```diff
@@ -12,7 +12,7 @@
 org.gradle.jvmargs=-Xmx1536m
 
 android.useAndroidX=true
-android.enableJetifier=false
+android.enableJetifier=true
```

Comment:
We don't use jetifier in avito app.
There are two options:
- Jetify krop manually for avito (see dependencies file)
- Fix the problem on a krop side (update libs or whatever need to be done to disable jetifier)

Reply:
Ok. I had no success at playing with gradle deps without jetify flag usage. 
Old color picker lib is outdated and no longer supported. 
We should just move to another picker.